src/unixTerminal.ts

```diff
@@ -171,7 +174,12 @@ export class UnixTerminal extends Terminal {
       if (err) {
         console.log('pty write error', err);
       }
+      console.log('written', written);
     });
+    // TODO: Understand why this hangs the process
+    // this._writeSocket.write(data, (err) => {
```

Comment:
Hi there! Is it possibly because the signature is:

```
socket.write(data[, encoding][, callback])
```

C.f. https://nodejs.org/api/net.html#socketwritedata-encoding-callback